README.md

```diff
@@ -766,6 +775,19 @@ Other Style Guides
     function handleThings(opts = {}) {
       // ...
     }
+    
+    // best
+    function isNullOrUndefined(obj){
+       // '==' handles both undefined and null
+       return obj == null;
+    }
+    function getDefaultValueIfNotPresent(obj,defaultValue){
+        return isNullOrUndefined(obj) ? defaultValue : obj;
+    }
```

Comment:
```suggestion
    function getDefaultValueIfNotPresent(obj,defaultValue) {
        return isNullOrUndefined(obj) ? defaultValue : obj;
    }
```
this is entirely unnecessary, though, since `??` exists - you can just do `obj ?? defaultValue`, without any need for a function.